linode/manager
# change: [M3-8384] - Update storybook to fix dependency vulnerabilities

## Description 📝
Small PR to update storybook to get rid of a high severity [dependabot alert](https://github.com/linode/manager/security/dependabot/101)

This update seems to be actually pretty major for storybook authors trying to clean up dependencies, which has been an ongoing problem. Hopefully this will help in the future.

## Changes  🔄
- Update storybook and related packages to `8.2.9`

## How to test 🧪

### Verification steps
- Run/build storybook locally to confirm no regression was introduced with this update

## As an Author I have considered 🤔

*Check all that apply*
